dashboard/admin.py

```diff
@@ -115,8 +116,14 @@ def save_model(self, request, obj, form, change):
         return super(CourseAdmin, self).save_model(request, obj, form, change)
 
 class LogResource(resources.ModelResource):
+
+    # This field is used to get the username from the user model
+    username = Field(attribute='user__username', column_name='username')
     class Meta:
         model = Log
+        # Show the same fields in the UI
+        fields = ('id', 'timestamp', 'username', 'action', 'extra')
```

Comment:
Just curious, the `id` field is not being shown from Django admin console. this is not related to this change, current prod is also the say way. 

we should still pull the `id` field as part of the this? or if we remove it breaks anything?

Reply:
I was debating leaving it out of the CSV and just having it match the UI but it feels like in an export having this column makes the data a little easier to deal with. I'm fine either way, maybe a @jennlove-um question.

Reply:
Ok, I feel we can leave it, it make sense to have in the CSV export